pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -457,12 +446,13 @@ def read_mwa_corr_fits(
         self.antenna_positions = antenna_positions_ecef - self.telescope_location
 
         # make initial antenna arrays, where ant_1 <= ant_2
-        ant_1_array = []
-        ant_2_array = []
-        for i in range(self.Nants_telescope):
-            for j in range(i, self.Nants_telescope):
-                ant_1_array.append(i)
-                ant_2_array.append(j)
+        ant_1_array, ant_2_array = np.transpose(
```

Comment:
Very clever! Can you just add a quick comment explaining that this is faster than the naive approach? (Similar to what you have in the `Nants_data` calculation is fine.)